raiden_contracts/tests/test_deploy_script.py

```diff
@@ -1023,3 +1027,25 @@ def test_verify_monitoring_service_deployment_with_wrong_onchain_token_address()
             service_registry_address=EMPTY_ADDRESS,
             user_deposit_address=FAKE_ADDRESS,
         )
+
+
+def test_user_deposit_deployment_with_wrong_one_to_n_address() -> None:
+    """ ContractVerifier.verify_user_deposit_deployment raises on a wrong OneToN address """
+    user_deposit_constructor_arguments = [
+        "0xDa12Dc74D2d0881749CCd9330ac4f0aecda5686a",
+        UINT256_MAX,
+    ]
+    wrong_one_to_n_address = FAKE_ADDRESS
+    user_deposit_mock = MagicMock()
+    mock_token_address = MagicMock()
+    mock_token_address.call.return_value = user_deposit_constructor_arguments[0]
+    user_deposit_mock.functions.token.return_value = mock_token_address
+    with pytest.raises(RuntimeError):
+        _verify_user_deposit_deployment(
+            user_deposit=user_deposit_mock,
+            constructor_arguments=user_deposit_constructor_arguments,
+            token_address=user_deposit_constructor_arguments[0],
+            user_deposit_whole_balance_limit=UINT256_MAX,
+            one_to_n_address=wrong_one_to_n_address,
+            monitoring_service_address="0xb7765972d78B6C97bB0a5a6b7529DC1fb64aA287",
+        )
```

Comment:
`user_deposit_constructor_arguments[0]` appears twice in this test and I had to check twice what that value was. It might be the heat, but perhaps rewriting it to this would be more straightforward to read:
```python
    user_deposit_wallet_addr = "0xDa12Dc74D2d0881749CCd9330ac4f0aecda5686a"
    user_deposit_constructor_arguments = [
        user_deposit_wallet_addr,
        UINT256_MAX,
    ]
    ...
    mock_token_address.call.return_value = user_deposit_wallet_addr
    ....
    with pytest.raises(RuntimeError):
        _verify_user_deposit_deployment(
            ...
            token_address=user_deposit_wallet_addr,
            ...
        )